relm4/src/factory/handle.rs

```diff
@@ -22,7 +22,6 @@ pub(super) struct FactoryHandle<C: FactoryComponent> {
 impl<C: FactoryComponent> fmt::Debug for FactoryHandle<C> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_struct("FactoryHandle")
-            .field("data", &self.data)
```

Comment:
We may want to leave this as `.field("data", &"<component>")` or similar instead.

Reply:
Done

Reply:
LGTM!